examples/simple-network/config-velocity.yml

```diff
@@ -0,0 +1,18 @@
+# This is a simplified config where the rest of the
```

Comment:
These first couple files are just the example config for a server. The actual protocol fix is further down